AFQ/api/bundle_dict.py

```diff
@@ -287,7 +287,7 @@ def baby_bd():
         "ATR_L", "ATR_R", "CST_L", "CST_R", "CGC_L", "CGC_R",
         "CCH_L", "CCH_R", "FP", "FA", "IFO_L", "IFO_R", "ILF_L",
```

Comment:
I believe we also need to eliminate "CCH" here, as per https://github.com/yeatmanlab/pyAFQ/pull/1070

Reply:
I'm wrong, as @36000 just explained to me.